src/Special/SpecialWikiSearchDataStandard.php

```diff
@@ -0,0 +1,199 @@
+<?php
+
+namespace WikiSearch\Special;
+
+use HTMLForm;
+use PermissionsError;
+use SpecialPage;
+use Status;
+
+/**
+ * Implements Special:WikiSearchDataStandard.
+ */
+class SpecialWikiSearchDataStandard extends SpecialPage {
+    /**
+     * @var string The canonical location of the data standard
+     */
+    private string $dataStandardLocation;
+
+    public function __construct() {
+        parent::__construct( 'WikiSearchDataStandard', 'wikisearch-edit-data-standard', false );
+    }
+
+    /**
+     * @inheritDoc
+     * @throws PermissionsError
+     */
+    public function execute($subPage ) {
```

Comment:
code styling space

Reply:
For some reason there are no tabs in the files, only spaces.. also code styling change